libs/ui-lib-tests/cypress/support/interceptors.ts

```diff
@@ -169,7 +169,7 @@ const addClusterListIntercepts = () => {
   cy.intercept('GET', allClustersApiPath, (req) => {
     const fixture = hasWizardSignal('CLUSTER_CREATED') ? updatedList() : initialList;
     req.reply(fixture);
-  });
+  }).as('list-clusters');
```

Comment:
I have done quite a big refactoring in my branch for Day2 to `interceptors.ts`.
Since you're only adding aliases, I think it's best to revert this file in your branch and we can add the missing aliases later (I added some there too, but maybe not all of them)